### PR TITLE
8339787: Add some additional diagnostic output to java/net/ipv6tests/UdpTest.java

### PR DESCRIPTION
**Pull request by serhiysachkov**
---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8339787](https://bugs.openjdk.org/browse/JDK-8339787): Add some additional diagnostic output to java/net/ipv6tests/UdpTest.java (**Enhancement** - P4)

### Reviewers
 * [Daniel Fuchs](https://openjdk.org/census#dfuchs) (@dfuch - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/21066/head:pull/21066` \
`$ git checkout pull/21066`

Update a local copy of the PR: \
`$ git checkout pull/21066` \
`$ git pull https://git.openjdk.org/jdk.git pull/21066/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 21066`

View PR using the GUI difftool: \
`$ git pr show -t 21066`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/21066.diff">https://git.openjdk.org/jdk/pull/21066.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/21066#issuecomment-2358795771)

### CODE REVIEW

File: test/jdk/java/net/ipv6tests/Tests.java [hunk 1]
```diff
@@ -123,7 +123,8 @@ public static void datagramEcho (DatagramSocket s1, DatagramSocket s2,
         }
 
         dprintln ("dest2 = " + dest2);
-
+        dprintln ("sender endpoint == " + s1.getLocalSocketAddress());
+        dprintln ("echo endpoint == " + s2.getLocalSocketAddress());
```
Comment (dfuch) [inline test/jdk/java/net/ipv6tests/Tests.java:127]: Nit: Any reason to have a single " = " line 125 and two " == " on these two lines?

Comment (serhiysachkov) [reply test/jdk/java/net/ipv6tests/Tests.java]: not really, changes to single = as in the rest of outputs

**Review by dfuch — APPROVED** [review]
Trivial and looks reasonable.

**Review by dfuch — CHANGES_REQUESTED** [review]
Actually - please update Copyright years too.

**Review by dfuch — APPROVED** [review]

**Pull request closed without merging**